p2p/shred.md

```diff
@@ -0,0 +1,313 @@
+# Shred packets
+
+The Solana Turbine protocol propagates confirmed data (blocks) within peer-to-peer networks in the form of shred packets.
```

Comment:
```suggestion
Shred packets are fragments of blocks (transaction data) that facilitate propagation of the blocks within peer-to-peer networks.
```
Feel free to tweak more as you see fit; some justification for the change
- I dropped confirmed as at the initial turbine broadcast I'd consider the status of the block to be "proposed"
- I dropped mention of turbine since we also transmit shreds around with repair (I think we consider repair to be distinct from turbine)